cumulusci/robotframework/Salesforce.py

```diff
@@ -492,6 +493,38 @@ def salesforce_insert(self, obj_name, **kwargs):
         self.store_session_record(obj_name, res["id"])
         return res["id"]
 
+    def salesforce_collection_insert(self, objects):
+        def dict_to_insertable(d):
+            insertable = {"attributes": {"type": d["type"]}}
+            for key, value in d.items():
+                if key != "type":
+                    insertable[key] = value
+            return insertable
+
+        insertables = [dict_to_insertable(o) for o in objects]
+
+        return self.cumulusci.sf.restful(
+            "composite/sobjects",
+            method="POST",
+            json={"allOrNone": False, "records": insertables},
+        )
+
+    def salesforce_init_objects(self, obj_name, number_to_create, **fields):
+        """Create an array of dictionaries with template-formatted arguments"""
```

Comment:
Why not expand the docstring to explain which names will be formatted and show an example? This will be included in the documentation generated by the robot_libdoc task.

Reply:
c1a48814c7aec5a330829373abc517ee6f0ea1ca